pythia/modules/layers.py

```diff
@@ -108,6 +111,37 @@ def forward(self, x):
         return logits
 
 
+class LanguageDecoder(nn.Module):
```

Comment:
Maybe move this to `decoders.py` as it is a decode.

Reply:
I added it here since this is a `ClassifierLayer` type module and other classifier layers are also in this file.